.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,61 @@
+# Workflow for publishing archives to PyPI
+name: Publish to PyPI
+
+on:
+  # Runs for pull requests for testing purposes
+  pull_request:
+    branches:
+      - master
+  # Only runs for releases
+  release:
+    types:
+      - published
+
+jobs:
+  publish-pypi:
+    name: Publish to PyPI
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2.3.4
+      with:
+        # fecth all history so that versioneer works
+        fetch-depth: 0
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+
+    - name: Install dependencies
+      run: python -m pip install setuptools wheel
+
+    - name: Fix up version string
+      run: |
+        # Change the versioneer format to "pre" so that the commit hash isn't
+        # included (PyPI doesn't allow it). Can't do this permanently because
+        # we rely the hash to indicate to the tests that this is a local
+        # verison instead of a published version.
+        #
+        # The step is only necessary for testing purpose
+        sed --in-place "s/pep440/pep440-pre/g" setup.cfg
```

Comment:
We may want to skip this step, as it's not needed for real releases.

Reply:
Can we ask @leouieda (who has more experience) to weigh in on this?

Reply:
I believe keeping it or not makes no difference to tagged releases, as the version strings would always be "vX.Y.Z" with no suffix string.

Reply:
Yep, this step doesn't make a difference for the actual releases since the version number generated is the same. We just merged a similar workaround for setuptools_scm in [fatiando/boule#61](https://github.com/fatiando/boule/pull/61/files#diff-28802fbf11c83a2eee09623fb192785e7ca92a3f40602a517c011b947a1822d3R70)